packages/react-scripts/template/src/service-worker-registration.js

```diff
@@ -0,0 +1,38 @@
+export default function register() {
```

Comment:
Apologies if I'm misunderstanding something about the project organization, but doesn't `react-dev-utils` contain a bunch of code that is consumed by `node`, and used optionally at build time? This particular code is consumed by the browser at runtime, and the hope is that most developers would end up using it.

There is the option of excluding registration, or explicitly switching to `unregister()` if needed (which is explained in the documentation), so putting this code in a place that's visible to the developer isn't necessarily a bad thing.